client/src/components/App/Content/InvestigationForm/TabManagement/ExposuresAndFlights/FlightsForm/AirportInput/AutoCompletedField/AutocompletedField.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import {TextField} from "@material-ui/core";
+import {Autocomplete, AutocompleteRenderInputParams} from "@material-ui/lab";
+import AutocompletedFieldType from "./AutoCompletedFieldTypes";
+import useStyles from "./AutoCompletedFieldStyles";
+import useFormStyles from "styles/formStyles";
+
+const AutocompletedField: AutocompletedFieldType = (props) => {
+    const {value, options, onChange, onInputChange, label, constOptions = false} = props;
+    const formClasses = useFormStyles();
+    const classes = useStyles();
+    const noOptionsMessage = 'וודאו כי הקלדתם שם תקין';
+
+    const inputElement = (params: AutocompleteRenderInputParams) =>
+        <TextField {...params}
+                   inputProps={{style: {padding: 0}, ...params.inputProps}}
+                   InputLabelProps={{classes: {root: formClasses.roundedTextLabel}}}
+                   InputProps={{
+                       classes: {root: formClasses.roundedTextField},
+                       ...params.InputProps
+                   }}
+                   label={label} variant="outlined" fullWidth/>;
+
+    const filterOptions = (x: any) => x;
```

Comment:
is this a prop that needs to be sent to Autocomplete? looks weird

Reply:
it is, because if the options come dynamicaly from an api and not froma const - you dont want to filter them you want to take everything - so filter will always be true

Reply:
cool, just wanted to understand it :)